ip_geoloc.module

```diff
@@ -75,32 +75,9 @@ function ip_geoloc_help($path, $arg) {
 /**
  * Implements hook_library().
```

Comment:
S/b `hook_library_info()`

Reply:
I see, but the PR and that module work for me despite this wrong hook. 
Has this to do with the backdrop_drupal_compatibility setting? If I switch that off, there are other errors which come up immediately, so this would need more work and it is another issue.

Reply:
Sorry, I think I was unclear. The docblock just needs to be updated to match the correct hook as you have fixed the function itself.

Reply:
Now I got it.